src/context/whoami/WhoAmI.tsx

```diff
@@ -20,6 +20,12 @@ export class WhoAmI {
     return this.me.displayName;
   }
 
+  public getRealm(): string {
+    if (this.me === undefined) return "";
+
+    return this.me.realm;
```

Comment:
This code can be shortened using [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).
```suggestion
    return this.me?.realm ?? "";
```